modules/gitops_configuring_sensitive_annotations_in_the_argo_cd_web_ui.adoc

```diff
@@ -0,0 +1,44 @@
+// Module is included in the following assemblies:
+//
+// * securing_openshift_gitops/masking-sensitive-annotations-in-the-argo-cd-web-ui.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="gitops_configuring_sensitive_annotations_in_the_argo_cd_web_ui_{context}"]
+= Configuring sensitive annotations in the Argo CD Web UI
```

Comment:
Can we change the title to ?
```suggestion
= Configuring Sensitive Annotations Masking in Argo CD Web UI and CLI
```

Reply:
Fixed. Hey @svghadi, as per the rules defined for using [Titles](https://www.ibm.com/docs/en/ibm-style?topic=grammar-capitalization#headings-titles-and-banners) in the IBM style guide, we use sentence-style capitalization for headings. So, changing it to headline-style capitalization is not as per our standards.

Reply:
Okay. What do you think about the suggested rephrasing?

Reply:
Hi @svghadi, how about this title: Masking Sensitive Annotations in Argo CD Web UI and CLI

Reply:
Sounds good.

Reply:
Hey @svghadi, I just realized I used a similar title in the Assembly file. We cannot use a similar one for the procedure. Here are two titles that I came up with. Please let me know which one do you prefer:

1. Enabling sensitive annotations masking in Argo CD UI and CLI
2. Configuring annotations masking for sensitive data in Argo CD UI and CLI

Reply:
1. looks good.